src/tlo/methods/healthburden.py

```diff
@@ -529,8 +543,61 @@ def log_df_line_by_line(key, description, df, force_cols=None) -> None:
             df=self.get_dalys(yld=yld_by_wealth, yll=yll_by_wealth),
             force_cols=self._causes_of_dalys,
         )
-
+        # 5) Log the prevalence of each disease
+        log_df_line_by_line(
+            key='prevalence_of_diseases',
+            description='Prevalence of each disease.',
+            df=self.prevalence_of_diseases,
+            force_cols=self.recognised_modules_names,
+        )
```

Comment:
is this  unintentionally left-in? The function is defined below (embedded in write-to-log-prevalence)